Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -340,6 +340,9 @@ def fetch_indicators_command(client, indicator_type, limit: Optional[int] = None
             for item in feed_dicts:
                 raw_json = dict(item)
                 raw_json['value'] = value = item.get('Name')
+                if value in indicators_value_set:
+                    continue
```

Comment:
if you will add an existing to a set, nothing will change. what am I missing here?

Reply:
but it will save run time